site/content/docs/user/loadbalancer.md

```diff
@@ -35,10 +35,13 @@ is not working anymore.
 kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.7/config/manifests/metallb-native.yaml
 {{< /codeFromInline >}}
 
-Wait for metallb pods to have a status of Running
+Wait until MetalLB is ready:
```

Comment:
can we be more specific? is the opd with the metallb controller, no?

Reply:
makes sense to me